src/app/roles.tsx

```diff
@@ -13,12 +13,18 @@ type RolesInterface = {
     [key in Roles]: string[];
 };
 
-const pathsShownToCaller = ["/calendar", "/clients", "/parcels", "/update-password"];
+const pathsShownToCaller = [
+    "/calendar",
+    "/clients",
+    "/parcels",
+    "/update-password",
+    "/set-password",
```

Comment:
should signed-in users be able to see /set-password at all?

Reply:
No they shouldn't. will remove

Reply:
actually need to keep this in - if a logged in user accesses this page, they should be logged out, but allowed to continue